cluster_pack/filesystem.py

```diff
@@ -148,7 +148,7 @@ def _genline(self) -> Iterator[bytes]:
             if out:
                 yield out
             else:
-                raise StopIteration
+                return

```

Comment:
What was the issue here ?
The best practise with generator functions is to raise this exception so that the code consuming the iterator knows what to do

Reply:
This was not working in python3.9. https://peps.python.org/pep-0479/

Reply:
agreed